content/kit/webcam.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Webcam
+---
+
+To see the outside world, a webcam is used to capture images for them to be processed. The webcam in the kit is a [TeckNet C016](http://www.tecknet.co.uk/c016.html).
+
+![The Webcam](http://www.tecknet.co.uk/media/catalog/product/cache/1/thumbnail/9df78eab33525d08d6e5fb8d27136e95/c/0/c016_webcam_4_.jpg?width=30pc)
+
+
+## Focus
+The webcam has a manual focus ring around the lens. During standard operation, this lens may be knocked, and move the image out of focus, which can seriously reduce the accuracy of the [vision](/api/vision/). 
+
+It's recommended to manually focus the camera using a separate computer, and then use a __non permanent__ method (hot glue is great for this)  
+
+{{% notice warning %}}
+There's no guarantee the webcam is correctly focused in your kit!
+{{% /notice %}}
+
+## Mounting the camera
+The camera can be mounted in many different ways, from cable ties to duct tape. We recommend using the 1/4" screw mount on the base of the camera, commonly used for tripods. This has the advantage of being far sturdier than other methods and less prone to moving, whilst reducing any risk of damaging the camera. 
```

Comment:
IMO part of the challenge is for them to figure this sort of thing out for themselves, so I'm not sure whether we should have this paragraph.

I can see the value in highlighting the screw mount, though I doubt they'll be able to use it as it's a left handed thread.